docs/theme-context.rst

```diff
@@ -0,0 +1,153 @@
+Read the Docs data passed to Sphinx build context
+=================================================
+
+Before calling `sphinx-build` to render your docs, Read the Docs injects some
+extra context in the templates by using the `html_context Sphinx setting`_ in the ``conf.py`` file.
+This extra context can be used to build some awesome features in your own theme.
+
+.. _html_context Sphinx setting: http://www.sphinx-doc.org/en/stable/config.html#confval-html_context
+
+.. note::
+
+   The `Read the Docs Sphinx Theme`_ uses this context to add additional features to the built documentation.
+
+.. _Read the Docs Sphinx Theme: https://sphinx-rtd-theme.readthedocs.io/en/latest/
+
+Context injected
+----------------
+
+Here is the full list of values injected by Read the Docs as a Python dictionary.
+Note that this dictionary is injected under the main key `readthedocs`:
+
+
+.. code:: python
```

Comment:
Add a comment about where this info came from / how to update it

Reply:
This code will be injected (when this docs get coded) at this point: https://github.com/rtfd/readthedocs.org/blob/0c547f47fb9dffbeb17e4e9ccf205a10caf31189/readthedocs/doc_builder/backends/sphinx.py#L65

... and there is no way to "update" RTD values since they are injected at the bottom of the `conf.py` after the values from the user.

So, do you think we should add that line of code to the docs? or you were thinking in another thing and I didn't get it? :)

Reply:
I was saying to add code comment for in the future we know where to generate the dictionary from.

Reply:
If I understood correctly, I already done this. Check my latest commit :)